### PR TITLE
new tailwindcss and alpinejs front-end

### PR DESCRIPTION
**Pull request by stesla**
I produced `public/market.json` by scraping the data on the current site, so that I could get the front-end going before we got the back-end going.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "description": "market-site\r ===========",
+  "main": "index.js",
+  "scripts": {
+    "build:css": "tailwindcss build css/styles.css -o public/styles.css",
+    "build:css:purge": "env NODE_ENV=production tailwindcss build css/styles.css -o public/styles.css",
+    "start": "live-server public --no-browser --port 8080"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/stesla/market-site.git"
```
Comment (lizthegrey) [inline package.json:13]: use eve-val instead through here

File: public/market.json [hunk 2]
```diff
@@ -0,0 +1,2866 @@
+[
```
Comment (lizthegrey) [inline public/market.json:1]: don't check this file in; this will be autogenerated by the python code in the parent dir

**Comment by stesla** [comment]
Removed market.json from history and fixed the URL. Also incorporated some UI feedback I got from one my team's designer after showing off screenshots.

File: package.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    "build:css:dist": "env NODE_ENV=production tailwindcss build css/styles.css -o public/styles.css",
+    "start": "live-server public --no-browser --port 8080"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eve-val/market-site"
+  },
+  "author": "Sam Tesla <samuel.tesla@gmail.com>",
+  "license": "ISC",
+  "bugs": {
```
Comment (lizthegrey) [inline package.json:17]: Same problem here with the URLs

**Review by lizthegrey — APPROVED** [review]

File: package.json (hunk 3 above)
Comment (stesla) [reply package.json:17]: Hrm. What should we put there?

Comment (stesla) [reply package.json:17]: I suppose we can probably omit author, tbh. But do we want something for license?

**Comment by lizthegrey** [comment]
https://github.com/eve-val/market-site/blob/master/LICENSE already MIT license

**Comment by stesla** [comment]
Of course it is. I'm pretty sure my eyes just glazed over it when I was filling out package.json. Fixed it in the first commit so as not to have it wrong at any point.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    "build:css:dist": "env NODE_ENV=production tailwindcss build css/styles.css -o public/styles.css",
+    "start": "live-server public --no-browser --port 8080"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/eve-val/market-site"
+  },
+  "author": "Sam Tesla <samuel.tesla@gmail.com>",
+  "license": "MIT",
+  "bugs":
```
Comment (lizthegrey) [inline package.json:17]: invalid json here, needs a `{`?

**Review by lizthegrey — CHANGES_REQUESTED** [review]

**Comment by stesla** [comment]
Okay, I added a `--json` flag that works just like the `--text` flag and outputs to stdout. Tested the result with the front-end, works like a charm.

**Comment by lizthegrey** [comment]
> Okay, I added a `--json` flag that works just like the `--text` flag and outputs to stdout. Tested the result with the front-end, works like a charm.

Almost there! One last change request: change the shell script to create both LSC4-P.html and market.json in the public dir and then cp them (and the css/js files) over to /srv/

**Review by lizthegrey — APPROVED** [review]

**Pull request merged**